tests/functional/test_data_completeness.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from botocore.session import Session
+from botocore.loaders import Loader
+from botocore.exceptions import DataNotFoundError
+
+
+class TestDataTypes(object):
```

Comment:
It is encapsulation, one of the 3 benefits of OO.

TL;DR.

The pattern of such test generator always include a generator and one (or more) helper. The helper(s) is tightly coupled with the generator, and is unlikely to be reused by others. The class encapsulation makes it explicit that these 2 guys always sit together. The conceptual benefit may not be obvious here but, should this file evolve to contain more test cases in future, I believe cases grouped by several classes would look better than [a bunch of flattened functions like this](https://github.com/boto/botocore/blob/develop/tests/integration/test_smoke.py#L181-L249).